metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/TableSequenceIds.java

```diff
@@ -26,4 +28,33 @@
 public class TableSequenceIds {
```

Comment:
Is `@Data` really needed here?

Reply:
Are you suggesting to just use @Getter.

Reply:
This class seems not to be immutable, we can use `@Value` or just `@getter` and related constructors.